Cargo.toml

```diff
@@ -71,7 +71,7 @@ spl-token = "3.5.0"
 reqwest = {version = "0", default-features=false, features = ["gzip", "json", "rustls-tls"]}
 beacon = { git = "https://github.com/helium/proto", branch = "master" }
 humantime = "2"
-metrics = "0.21"
+metrics = "0.22"
```

Comment:
should this be `>=0.22` ?

Reply:
Indeed it does, I forgot cargo treats pre `1.0` versions differently.